Content.Shared/DoAfter/SharedDoAfterSystem.Update.cs

```diff
@@ -188,7 +188,7 @@ private bool ShouldCancel(DoAfter doAfter,
         {
             if (args.DistanceThreshold != null)
             {
-                if (!_interaction.InRangeUnobstructed(args.User, args.Target.Value, args.DistanceThreshold.Value))
+                if (!_interaction.InRangeAndAccessible(args.User, args.Target.Value, args.DistanceThreshold.Value))
```

Comment:
Okay so actually this breaks something in a kinda weird way:

Admin ghosts can still see the strip window even if the target is in a container. Now *for some reason* the admin ghost can't strip things worn by the target, but it *can* take things from their hands. This PR breaks the hand taking.

I think maybe the answer is just "aghosts should never have to worry about ShouldCancel" (and really they should be able to strip worn objects as well), but yeah that needs to be fixed. 

Otherwise I found no issues with this PR.